l10n_br_fiscal/tools.py

```diff
@@ -89,3 +89,47 @@ def remove_non_ascii_characters(value):
         )
 
     return result
+
+
+def set_journal_in_fiscal_operation(cr, company, values):
+    """
+    Set Journal in Fiscal Operation by 'ir.property'
+    :param company: Company Object
+    :param values: Dict with Journal and Fiscal Operation
+    """
+    _logger.info(
+        f"Create or Inform Journal in Fiscal Operation for {company.name} Property ..."
+    )
+    for value in values:
+        env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
please move the env creation outside of the loop (optimization)

Reply:
Done